WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.kt

```diff
@@ -717,6 +718,7 @@ class EditPostActivity : LocaleAwareActivity(), EditorFragmentActivity, EditorIm
                 }
 
             isNewPost = state.getBoolean(EditPostActivityConstants.STATE_KEY_IS_NEW_POST, false)
+            isVoiceContentSet = state.getBoolean(EditPostActivityConstants.STATE_KEY_IS_NEW_POST, false)
```

Comment:
`isVoiceContentSet` uses the wrong key here, it should use `STATE_KEY_IS_VOICE_CONTENT_SET`.

Reply:
Updated in [82a6ca6](https://github.com/wordpress-mobile/WordPress-Android/pull/20992/commits/82a6ca668a10a96036538c15cabfba43d8769a30)